### ISSUE TITLE
onValueChange called initially on Android

### ISSUE THREAD

**Issue by jenskuhrjorgensen** [lesliesam/react-native-wheel-picker#41]
On Android, the `onValueChange` callback is called upon instantiation. On iOS it doesn't happen. 

Thanks for the lib :)

**Comment by sefam-vbastien** [comment]
Hello,

After some investigation this seems to be due to the underlaying wheel picker library. It does emit value at the 'onSizeChanged' of the view event and at the 'run' of the Runnable of this library.

We did try to mimic the ios library which does not emit any value before the user does press the wheel.

To do so we did the following code available here: https://github.com/sefam/react-native-wheel-picker/commit/64f4a9dbdb70030c1864961dd14534b756b65965 (Note that we did not make a pull request since there is still an issue we are ok with)

The main things is to abort all value emissions until the view is ready. We assume it is when the user press the view or does a gesture. We put a boolean to check this and emit only then.

The only emission exception is to mimic the ios behavior. The ios component emit at first touch even if the user does not change the displayed value (e.g the initial value is 120 , the user move the wheel a bit but goes back to 120). To do so we did a hack using onTouchEvent that emit the last value stored.
This works well except that if the user change the value at first time you got 2 events (the initial value and the new one)

If you guys have some better approach to solve this issue we will be happy to hear it.
